tests/models/Entry_test.js

```diff
@@ -0,0 +1,280 @@
+'use strict';
+//TEST SETUP
+let Entry;
+let assert = chai.assert;
+let GetEntry = () =>  {
+  return {
+    id: 1,
+    title: 'Test Entry',
+    guid: '1',
+    link: 'Test Link 1',
+    updated: new Date(),
+    content: 'Test Content 1',
+    content_hash: 'Test Content Hash 1',
+    cached_content: 'Test Cached Content 1',
+    date_entered: new Date(),
+    date_updated: new Date(),
+    lang: 'English'
+  };
+};
+
+describe('Entry', function() {
+  co.wrap(function* () {
+    let sequelize = yield getTestDB();
+    Entry = sequelize.models.Entry;
+  })();
+  it('Saves and retrieves successfully', co.wrap(function* () {
+    yield Entry.sync({force: true});
```

Comment:
I was under the impression that this would clear the table at the start of every test.